lib/doorkeeper/models/access_grant_mixin.rb

```diff
@@ -28,7 +28,7 @@ module AccessGrantMixin
 
     module ClassMethods
       def by_token(token)
-        where(token: token.to_s).limit(1).to_a.first
```

Comment:
This were introduced in https://github.com/doorkeeper-gem/doorkeeper/pull/484, as performance improvements by @mtparet. Do you know if `find_by` performs an `ORDER` operation?